### PR TITLE
Add a job to check import formatting using isort

### PR DESCRIPTION
**Pull request by IRDonch**
### Summary
This allows us to automatically enforce consistent code style when it comes to imports.

I fixed all existing inconsistencies by letting `isort` reformat the entire codebase.

### How to test

N/A

### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added description of my changes into [CHANGELOG](https://github.com/openvinotoolkit/datumaro/blob/develop/CHANGELOG.md)
- [ ] I have updated the [documentation](
  https://github.com/openvinotoolkit/datumaro/tree/develop/docs) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have [linked related issues](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/openvinotoolkit/datumaro/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2021 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

**Review by yasakova-anastasia — APPROVED** [review]

File: datumaro/cli/__main__.py [hunk 1]
```diff
@@ -8,10 +8,9 @@
 import os.path as osp
 import sys
 
-from . import contexts, commands
-from .util import CliException, add_subparser
+from . import commands, contexts
 from ..version import VERSION
-
+from .util import CliException, add_subparser
```
Comment (zhiltsov-max) [inline datumaro/cli/__main__.py:13]: I'm not sure I understand the order it produces. What are the rules for `import` and `from import`?

File: datumaro/cli/commands/merge.py [hunk 2]
```diff
@@ -3,16 +3,16 @@
 # SPDX-License-Identifier: MIT
 
 import argparse
+from collections import OrderedDict
 import json
 import logging as log
 import os.path as osp
```
Comment (zhiltsov-max) [inline datumaro/cli/commands/merge.py:9]: I'm not sure I understand the order it produces. What are the rules for `import` and `from import`?

File: .isort.cfg [hunk 3]
```diff
@@ -0,0 +1,6 @@
+[settings]
+force_sort_within_sections = True
```
Comment (zhiltsov-max) [inline .isort.cfg:2]:
```suggestion
from_first = True
```

**Review by zhiltsov-max — CHANGES_REQUESTED** [review]
Nice PR!

File: .isort.cfg (hunk 3 above)
Comment (IRDonch) [reply .isort.cfg]: OK, updated.

**Review by zhiltsov-max — APPROVED** [review]

**Pull request merged**